apps/tlon-mobile/cosmos.imports.ts

```diff
@@ -62,7 +62,7 @@ import * as fixture54 from './src/fixtures/ActionSheet/AddGalleryPostSheet.fixtu
 import * as decorator0 from './src/fixtures/cosmos.decorator';
 
 export const rendererConfig: RendererConfig = {
-  "playgroundUrl": "http://localhost:5001",
+  "playgroundUrl": "http://localhost:5000",
```

Comment:
intentional?

Reply:
happened automatically when running cosmos, afaik 5000 is the default. I'm guessing someone previously ran cosmos while also running another service that was bound to port 5000, then cosmos.imports.ts was updated with that value and it got committed that way.